app/src/test/java/ch/sbb/polarion/extension/generic/configuration/ConfigurationStatusProviderTest.java

```diff
@@ -19,7 +20,7 @@ class ConfigurationStatusProviderTest {
     void testGetAllStatuses() {
         try (MockedStatic<ContextUtils> contextUtilsMockedStatic = mockStatic(ContextUtils.class)) {
             contextUtilsMockedStatic.when(() -> ContextUtils.findSubTypes(any())).thenReturn(Set.of(TestSingleProvider.class, TestMultipleProvider.class));
-            List<ConfigurationStatus> statuses = ConfigurationStatusProvider.getAllStatuses("some scope");
+            Collection<ConfigurationStatus> statuses = ConfigurationStatusProvider.getAllStatuses("some scope");
             assertEquals(3, statuses.size());
             assertTrue(statuses.containsAll(List.of(
```

Comment:
here we need to check not contains all, but the sequence in alphabetical order. could you please change the validation?